docs/source/theory/intro.rst

```diff
@@ -55,6 +55,38 @@ Unlike particles within the beam, the reference particle is described by a set o
 in a global laboratory coordinate system (x,y,z).  The momenta of the reference particle are normalized by :math:`mc`, so that :math:`p_x=\beta_x\gamma`, etc.  A parameteric plot of
 the reference trajectory variables (x,z) allows the user to view the global geometry of the accelerator structure (footprint).
 
+.. _theory-collective-beam-distribution-input:
+
+Beam Distribution Input
+-----------------------
+
+.. figure:: phase_space_ellipse.svg
```

Comment:
With 1.1kB, this svg seems small enough to check in (instead of adding to the PR description)